### ISSUE TITLE
Next starter kit should not be used for production?

### ISSUE THREAD

**Issue by adrienshen** [reactioncommerce/reaction-next-starterkit#487]
On the project README.md documentation, it saids that this project should not be used for production. Which means that it might not be stable yet.

Yet, on the ReactCommerce site, the documentation seems to point at using 2.0 and Next starter template seems to be the recommended approach based on that documentation over there.

This is somewhat confusing and concerning, and I am not sure what is stable to use in production, as well as provide the best developer experience.

What is the official recommended approached to customize the frontend as of January 21, 2019?

**Comment by spencern** [comment]
I've closed this with #543 which removes the production warning, but I'd also like to provide some additional context for this.

Tl;dr: this project, [Reaction Platform](https://github.com/reactioncommerce/reaction-platform), and the v2.0 version of Reaction in general is our recommend approach to building a shop with Reaction.

Longer answer:
We've been working toward version 2.0 of Reaction for almost 9 months. When we cut our first release candidate for 2.0, our example storefront (this repository) was not at feature parity with the Meteor storefront which was built into the our monolithic application. We've since removed the storefront entirely from the `reaction`  repo (See: https://github.com/reactioncommerce/reaction/pull/4948, https://github.com/reactioncommerce/reaction/pull/4947 as examples) and as of [v2.0.0-rc.11](https://github.com/reactioncommerce/reaction/releases/tag/v2.0.0-rc.11) this example storefront is the recommended way to build a storefront on top of Reaction Commerce.

This project represents an _example_ of how you could build a storefront to consume the headless Reaction Commerce GraphQL API. We haven't provided a lot of tooling to aid in customizing this example storefront. You're welcome to start with this project as a base for your own storefront, but this is certainly not a one-size-fits-all project. You should be able to build a storefront for Reaction that uses React, Angular, Ember, or Vue; Apollo or Relay; Styled Components, CSS-in-JS, or bootstrap; and Gatsby, Next.js, create-react-app, or rolling your own webpack config. We're not trying to provide an opinion about how you should approach your storefront with this project, but rather to provide an example of how you can consume the Reaction GraphQL API to create your own shop.

In fact, if you're interested in building your own storefront and open sourcing it, we'd love to promote it. We'd also love to see other approaches to building storefronts that work with our GraphQL API. We believe that the Reaction Commerce project will be stronger with an ecosystem of many storefronts which are developed by our community that provide a varied approaches to building a shop on Reaction. 

In support of this, we have [renamed](https://github.com/reactioncommerce/example-storefront/pull/544) this project from `reaction-next-starterkit` to `example-storefront` to better convey the intent behind our creating it.  We’re also [updating our docs](https://github.com/reactioncommerce/reaction-docs/pull/829) to clarify this change.
